src/datafactory/README.md

```diff
@@ -0,0 +1,5 @@
+Microsoft Azure CLI 'datafactory' Extension
+==========================================
+
+This package is for the 'datafactory' extension.
+i.e. 'az datafactory'

```

Comment:
Please add more description, like https://github.com/Azure/azure-cli-extensions/blob/master/src/storage-preview/README.md

Reply:
This has not been fixed yet, the outdated status is just because I changed the eof mark.